Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -549,10 +548,6 @@ - (void)viewWillAppear:(BOOL)animated
     // In case we're dismissing a CNContactViewController which requires default system appearance
     [UIUtil applySignalAppearence];
 
-    // Since we're using a custom back button, we have to do some extra work to manage the
-    // interactivePopGestureRecognizer
-    self.navigationController.interactivePopGestureRecognizer.delegate = self;
```

Comment:
I'm not clear why we were doing this since this class and its JSQ superclass don't actually implement any `UIGestureRecognizerDelegate` methods.  It was complicating and/or interfering with `OWSNavigationController`, so I've removed it.